<launch>
  <node name="orb_slam2_rgbd" pkg="orb_slam2_ros"
      type="orb_slam2_ros_rgbd" args="
          $(find orb_slam2_ros)/orb_slam2/Vocabulary/ORBvoc.txt
          $(find orb_slam2_ros)/orb_slam2/config/RealSenseR200RGBD.yaml"
      output="screen">
       <remap from="/camera/rgb/image_raw" to="/head_camera/rgb/image_raw" />
       <remap from="/camera/depth_registered/image_raw" to="/head_camera/depth_registered/image_raw" />

       <param name="publish_pointcloud" type="bool" value="true" />
       <param name="pointcloud_frame_id" type="string" value="map" />
       <param name="camera_frame_id" type="string" value="odom" />>
  </node>
</launch>
